frontend/src/app/modules/common/dynamic-forms/components/dynamic-field-wrapper/dynamic-field-wrapper.component.html

```diff
@@ -1,13 +1,14 @@
-<op-form-field
-  [label]="to?.label"
-  [noWrapLabel]="to?.noWrapLabel"
-  [required]="to?.required"
-  [helpTextAttribute]="to?.property"
-  [helpTextAttributeScope]="to?.helpTextAttributeScope"
-  [attr.data-qa-field-name]="to?.property"
-  [ngClass]="{ 'op-form-field_checkbox': field.type === 'booleanInput' }"
->
```

Comment:
Some time ago, we decided on this template style:

```
<element
    attr1=foo
    attr2=bar
>
   content
</element>
```

your IDE is probably not configured to apply this correctly on reformatting.